.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
```

Comment:
After chatting with @philbrookes, we agreed to use the latest to pick up on any potential issues building with the latest version